packages/tools/webpack-fluid-loader/webpack.config.js

```diff
@@ -25,9 +28,19 @@ module.exports = {
       },
     ],
   },
+  // Webpack 5 does not support automatic polyfilling of node modules, setting node to false will help simulate webpack 5 behavior by throwing build errors when we rely on node polyfills
```

Comment:
I made webpack-fluid-loader act like webpack 5 so we don't regress this anymore by adding more node dependencies in browser code. The last blocking issue is #4040